packages/core/src/state-management/state-manager.ts

```diff
@@ -54,7 +56,9 @@ export class StateManager {
     private readonly load: (
       streamId: StreamID,
       opts?: LoadOpts | CreateOpts
-    ) => Promise<RunningState>
+    ) => Promise<RunningState>,
+    private readonly indexStreamIfNeeded,
```

Comment:
this would need a type specifier, except that I want to get it out of here entirely

Reply:
done - removed.